packages/fast-components-react-msft/src/paragraph/paragraph.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import { get } from "lodash-es";
+import { Foundation, HandledProps, TypeLevel, TypographyTag } from "@microsoft/fast-components-react-base";
+import {
+    IParagraphHandledProps,
+    IParagraphManagedClasses,
+    IParagraphUnhandledProps,
+    ParagraphLevel
+} from "./paragraph.props";
+import Typography from "../typography";
+import { IManagedClasses, IParagraphClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+
+/* tslint:disable-next-line */
+class Paragraph extends Foundation<IParagraphHandledProps & IManagedClasses<IParagraphClassNameContract>, React.HTMLAttributes<HTMLParagraphElement>, {}> {
+    protected handledProps: HandledProps<IParagraphHandledProps & IManagedClasses<IParagraphClassNameContract>> = {
+        level: void 0,
+        managedClasses: void 0
+    };
+
+    /**
+     * Renders the component
+     */
+    public render(): React.ReactElement<HTMLHeadingElement | HTMLParagraphElement> {
+        return (
+            <Typography
+                {...this.unhandledProps()}
+                tag={TypographyTag.p}
+                typeLevel={this.getTypeLevelFromProps()}
+                className={this.generateClassNames()}
+            >
+                {this.props.children}
+            </Typography>
+        );
+    }
+
+    /**
+     * Generates class names based on props
+     */
+    protected generateClassNames(): string {
+        const classes: string = get(this.props, `managedClasses.paragraph_${this.props.level || ParagraphLevel._3}`);
+
+        return super.generateClassNames(classes);
+    }
+
+    /**
+     * Stores level for use in render
+     */
+    private getTypeLevelFromProps(): TypeLevel {
```

Comment:
naming nit: I don't think that either "get" and "fromProps" add any semantic value to this function name. It can be reduced to `typeLevel` and it still convey all necessary meaning.